api/src/main/java/org/apache/iceberg/catalog/SupportsNamespaces.java

```diff
@@ -100,7 +100,7 @@ default void createNamespace(Namespace namespace) {
    *
    * @param namespace a namespace. {@link Namespace}
    * @return true if the namespace was dropped, false otherwise.
-   * @throws NamespaceNotEmptyException If the namespace does not empty
+   * @throws NamespaceNotEmptyException If the namespace is not empty
```

Comment:
I was confused for a second what `is not empty` really means and wondered whether the javadoc should be: `NamespaceNotEmptyException If the namespace is not empty and contains tables` or something along those lines

Reply:
I think it is pretty clear what empty means -- it has tables or nested namespaces. I think the "and contains tables" wording is actually less clear because it makes it sound like containing tables is distinct from being not empty, like a separate requirement. We could probably fix that up with "that is, if it contains tables or nested namespaces"... but again, I think it's reasonably clear. Maybe I'm wrong, so if anyone else agrees that we should clarify please reply!